### DISCUSSION TITLE
Antivirus Issues?

### DISCUSSION THREAD

**Discussion by JDraugr** [leecher1337/ntvdmx64#220] [Q&A] [answered]
Does using the auto build process give false positives to antivirus scanners?

I use BitDefender and got the following warnings.

The file C:\Users\John\AppData\Local\Microsoft\Windows\INetCache\IE\MI27191A\ntvdmx64-master[1].zip has been detected as infected with Gen:Variant.Tedy.121628. Bitdefender deleted this item, your device is safe.

I then tried to use the autobuild on a test PC in case the files were infected. The autobuild worked fine, so I ran the install.bat in the release folder and BitDefender reported...

The item C:\Windows\SysWOW64\ldntvdm.dll was deleted.

I'm wary to try to install this on my main PC. Has anyone else had these issues or know if these are normal false positives that anit virus programs detect?

Thanks,
John

**Comment by leecher1337** [comment, marked as answer]
Normal false positives, lots of tickets for it already, just tell your Antivirus to leave the files alone.
There is nothing I can do about stupid AV-programs. You may want to report false positive to your AV vendor so that it can be whitelisted in signatures, but as soon as loader gets updated, the whole false positive crap will surface again...

https://github.com/leecher1337/ntvdmx64/issues/126
https://github.com/leecher1337/ntvdmx64/issues/216
https://github.com/leecher1337/ntvdmx64/issues/10
https://github.com/leecher1337/ntvdmx64/issues/11
https://github.com/leecher1337/ntvdmx64/issues/22

PS: Gen. usually means "Generic signature", that means that there is no exact signature match and the match is pure speculation by the Antivirus, so such detections generally cannot really be taken too seriously.
